meta/main.yml

```diff
@@ -0,0 +1,15 @@
+galaxy_info:
+  author: David Roble
+  description: Changes the password for a user account
+  company: Red Hat, Inc.
+  license: GPLv3
+  min_ansible_version: 2.4
```

Comment:
Is this really the minimum version? We can probably roll much farther back if we wanted for this role.

Reply:
I don't mind either way. If our goal is to make all of OASIS 2.4+ and we don't have any customers clamoring for anything older, let's just leave it as is.

Reply:
:+1: for 2.4+